boards/arm/nrf52840_pca10059/nrf52840_pca10059.dts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840.dtsi>
+
+/ {
+	model = "Nordic PCA10059 Dev Kit";
+	compatible = "nordic,pca10059-dk", "nordic,nrf52840-qiaa",
+		     "nordic,nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+#ifdef CONFIG_MCUMGR_SMP_UART
+		zephyr,uart-mcumgr = &uart0;
+#endif
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+&gpiote {
+	status ="ok";
+};
+
+&gpio0 {
+	status ="ok";
+};
+
+&gpio1 {
+	status ="ok";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
```

Comment:
Could we use the newly developed UARTE driver?

Reply:
we don't really have a UART here. I think we can remove this.

Reply:
Agree, we should remove this.

Reply:
That's not possible, we _need_ the UART driver to use CDC ACM and we need this for the UART driver to compile.

Reply:
Could we use the UARTE driver, then?